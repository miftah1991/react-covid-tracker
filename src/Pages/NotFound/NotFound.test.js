import renderer from 'react-test-renderer';
import {
  render, screen,
} from '@testing-library/react';
import NotFound from './NotFound';

it('snapshot of homepage', () => {
  const tree = renderer.create(
    <NotFound />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('rendering page', () => {
  render(<NotFound />);
  expect(screen.getByText('NotFound')).toBeInTheDocument();
});
