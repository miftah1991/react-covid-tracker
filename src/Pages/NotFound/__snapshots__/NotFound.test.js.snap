// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of homepage 1`] = `
<div>
  NotFound
</div>
`;
